v20.2/sso.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Single Sign-on (Enterprise)
+summary: Implement single sign-on (SSO) for Admin UI access.
+toc: true
+---
+
+Single sign-on (SSO) allows a user to access the Admin UI in a secure cluster via an external identity provider. When SSO is configured and enabled, the [Admin UI login page](admin-ui-overview.html#admin-ui-access) will display an OAuth login button in addition to the password access option.
```

Comment:
Can we specify which user we are referring to? CockroachDB user, SQL user, or Google user? And is the same user being referred to throughout the doc?